roles/iommu/tasks/main.yml

```diff
@@ -1,14 +1,55 @@
 ---
+- name: Template dracut config for vfio
+  ansible.builtin.blockinfile:
+    path: /etc/dracut.conf.d/gpu-vfio.conf
+    block: |
+      add_drivers+="vfio vfio_iommu_type1 vfio_pci vfio_virqfd"
```

Comment:
I saw warnings about this configuration:

```
2024-08-27T12:52:11+0000 INFO /etc/dracut.conf.d/gpu-vfio.conf:add_drivers+="vfio vfio_iommu_type1 vfio_pci vfio_virqfd"

dracut: WARNING: <key>+=" <values> ": <values> should have surrounding white spaces!
dracut: WARNING: This will lead to unwanted side effects! Please fix the configuration file.

dracut-install: Failed to find module 'vfio_virqfd'
dracut: FAILED:  /usr/lib/dracut/dracut-install -D /var/tmp/dracut.kNu52l/initramfs --kerneldir /lib/modules/5.14.0-362.24.1.el9_3.0.1.x86_64/ -m vfio vfio_iommu_type1 vfio_pci vfio_virqfd
/etc/dracut.conf.d/gpu-vfio.conf:add_drivers+="vfio vfio_iommu_type1 vfio_pci vfio_virqfd"

dracut: WARNING: <key>+=" <values> ": <values> should have surrounding white spaces!
dracut: WARNING: This will lead to unwanted side effects! Please fix the configuration file.

dracut-install: Failed to find module 'vfio_virqfd'
dracut: FAILED:  /usr/lib/dracut/dracut-install -D /var/tmp/dracut.gebJKd/initramfs --kerneldir /lib/modules/5.14.0-362.24.1.el9_3.0.1.x86_64/ -m vfio vfio_iommu_type1 vfio_pci vfio_virqfd
```

Please fix the syntax.